README.md

```diff
@@ -103,13 +103,21 @@ or, if you want comments to appear on pull request diff, instead of commit:
 $ GITHUB_ACCESS_TOKEN=token PULL_REQUEST_ID=id pronto run -f github_pr -c origin/master
 ```
 
+Use `GithubStatusFormatter` to submit [commit status](https://github.com/blog/1227-commit-status-api):
+
+```sh
+$ GITHUB_ACCESS_TOKEN=token pronto run -f github_status -c origin/master
```

Comment:
Ok, I'll add one more example.
Although I believe using multiple formatters is not documented anywhere else.